patches/tModLoader/Terraria/CommonDocs.xml

```diff
@@ -27,4 +27,19 @@
 		<para/> Lighting is tracked at Tile Coordinate granularity (<see href="https://github.com/tModLoader/tModLoader/wiki/Coordinates">Coordinates wiki page</see>). Typically the center of an entity, converted to tile coordinates, is used to query the lighting color at the entity's location. That color is then used to draw the entity.
 		<para/> Lighting values in-between tile coordinates can be interpolated using <see cref="GetSubLight(Vector2)"/>, but usually the lighting values at the tile coordinates closest to the center of an entity are sufficient. There are many more GetColorX methods for more advanced situations, if needed. If drawing a really large sprite, one might consider splitting up the drawing to allow sections of the sprite to be drawn at different light values.
 	</LightingGetColor>
+	<BitArrayUsage>
+		<para/> Useful for sending/receiving large amounts of flags/bools. If the data length is unknown to the receiver, length would need to be sent as well.
+		This example sends a bool array of length <c>ItemLoader.ItemCount</c>, which will be consistent, so length is not sent.
+		<code>
+// Field
+bool[] itemsFound = new bool[ItemLoader.ItemCount];
+		
+// NetSend
+Utils.SendBitArray(new (itemsFound), writer);
```

Comment:
If we're going to promote these, we should make overloads which accept `bool[]` as the input and output. Including ones that don't alloc a new `bool[]` or `BitArray`.

Reply:
Deferred for now, as suggested.